modules/puppet_proxy/initializer.rb

```diff
@@ -1,4 +1,4 @@
-require 'puppet'
+require 'puppet' if RUBY_PLATFORM =~ /ruby/
```

Comment:
Just tested this again. Removing the conditionals form puppt-proxy leaves me with:
```
c:/smart-proxy/modules/puppet_proxy/initializer.rb:1:in `require': cannot load such file -- puppet (LoadError)
        from c:/smart-proxy/modules/puppet_proxy/initializer.rb:1:in `<top (required)>'
        from c:/smart-proxy/modules/puppet_proxy/puppet_plugin.rb:1:in `require'
        from c:/smart-proxy/modules/puppet_proxy/puppet_plugin.rb:1:in `<top (required)>'
        from c:/smart-proxy/modules/puppet_proxy/puppet.rb:1:in `require'
        from c:/smart-proxy/modules/puppet_proxy/puppet.rb:1:in `<top (required)>'
        from c:/smart-proxy/lib/smart_proxy.rb:63:in `require'
        from c:/smart-proxy/lib/smart_proxy.rb:63:in `<module:Proxy>'
        from c:/smart-proxy/lib/smart_proxy.rb:38:in `<top (required)>'
        from C:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from C:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from bin/smart-proxy:5:in `<main>'
```